roles/solr_collection/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+# tasks file for roles/solr_collection
+- name: Create the base directory for Solr configsets
+  file:
+    path: "/solr/cdh_solr/"
+    state: directory
+    mode: 0755
+    owner: "{{ deploy_user }}"
+    group: "{{ deploy_user }}"
+  become: true
+  delegate_to: lib-solr-staging1.princeton.edu  # solr_staging
+
+- name: Update and configure Solr configset
+  block:
+    - name: Copy project Solr configset files for upload to zookeeper
+      # rsync configset from deployed version on remote host to solr server
+      become: true
+      become_user: "{{ deploy_user }}"
+      command: /usr/bin/rsync -a cdh-test-geniza1:{{ deploy }}/solr_conf/ /solr/cdh_solr/cdh_{{ app_name }}_solr_conf
```

Comment:
Presumably this is easier than using ansible's built in rsync command? https://docs.ansible.com/ansible/latest/collections/ansible/posix/synchronize_module.html

Reply:
We couldn't get it to work! I initially tried using it and spent quite a while troubleshooting it when everything I tested manually worked fine, so finally we decided to switch to just using rsync directly.